### PR TITLE
Generalimprovements

### PR DESCRIPTION
**Pull request by jenniferlynparsons**

### CODE REVIEW

File: _pages/culture-and-process.md [hunk 1]
```diff
@@ -6,96 +6,45 @@ permalink: /culture-and-process
...
-that are like you, that share your understanding and concerns. They are places
-where you can talk about things that have happened or things you'd like to see
-without having to compensate for all the intersections that might be present in
-resource channels.
+Identity channels are places where you can get to know people in the community that are like you, that share your understanding and concerns. They are places where you can talk about things that have happened or things you'd like to see without having to compensate for all the intersections that might be present in resource channels.
 
-Identity channels fall into two categories: "toplevel" identity channels (for
-example, `#poc` or `#quiltbag`), and "subchannels" under those identities
-(`#poc-mental-health`, #lgbtq-politics`).
+Identity channels fall into two categories: "toplevel" identity channels (for example, `#poc` or `#quiltbag`), and "subchannels" under those identities (`#poc-mental-health`, #lgbtq-politics`).
```
Comment (varjmes) [inline _pages/culture-and-process.md]: #lgbtq-politics needs a starting "`"

i.e `#lgbtq-politics`

File: assets/css/style.css [hunk 2]
```diff
@@ -242,7 +242,7 @@ input[type=number]::-webkit-outer-spin-button {
 }
 .form-style-2 input.input-field{
     width: 48%;
-   
+
```
Comment (varjmes) [inline assets/css/style.css]: why the extra line break?

File: assets/css/style.css [hunk 3]
```diff
@@ -633,7 +633,7 @@ html {
 .blog-title {
   font-size: 72px;
   line-height: 72px;
-  text-transform: uppercase;
+  /*text-transform: uppercase;*/
```
Comment (varjmes) [inline assets/css/style.css]: can we get rid of this? :)

**Comment by varjmes** [comment]
A few minor changes and this is good to merge :)

File: _pages/culture-and-process.md (hunk 1 above)
Comment (jenniferlynparsons) [reply _pages/culture-and-process.md]: done!

File: assets/css/style.css (hunk 2 above)
Comment (jenniferlynparsons) [reply assets/css/style.css]: no idea where that came from. yay!

File: assets/css/style.css (hunk 3 above)
Comment (jenniferlynparsons) [reply assets/css/style.css]: deleted!

**Pull request merged**